core/base/CMakeLists.txt

```diff
@@ -198,11 +198,27 @@ set(BASE_SOURCES
   src/TVirtualX.cxx
 )
 
+set(FULL_BASE_HEADERS ${BASE_HEADERS})
+list(TRANSFORM FULL_BASE_HEADERS PREPEND ${CMAKE_CURRENT_SOURCE_DIR}/inc/)
+
```

Comment:
Reading the documentation in CMake, I believe this is unnecessary, since CMake will expand the paths relative to the current directory (+ a possible BASE_DIR), see below.
```suggestion
```